scripts/xsslint/tests/test_main.py

```diff
@@ -0,0 +1,179 @@
+# -*- coding: utf-8 -*-
+"""
+Tests for main.py
+"""
+import re
+import textwrap
```

Comment:
I think this is unused.  Maybe run isort on everything?

Reply:
Done.